firebase-database/src/androidMain/kotlin/dev/gitlive/firebase/database/database.kt

```diff
@@ -38,7 +38,7 @@ suspend fun <T> Task<T>.awaitWhileOnline(database: FirebaseDatabase): T =
             .reference(".info/connected")
             .valueEvents
             .debounce(2.seconds)
-            .filter { !it.value<Boolean>() }
+            .filterNot { it.value<Boolean>() }
```

Comment:
🥂

Reply:
To be fair, this is from the upstream :P